pulpcore/tasking/pulpcore_worker.py

```diff
@@ -162,8 +167,8 @@ def worker_cleanup(self):
 
     def beat(self):
         if self.worker.last_heartbeat < timezone.now() - timedelta(seconds=self.heartbeat_period):
-            self.worker = handle_worker_heartbeat(self.name)
-            if self.shutdown_requested:
+            self.worker = self.handle_worker_heartbeat()
```

Comment:
This line should not have changed.
```suggestion
            self.worker = handle_worker_heartbeat(self.name)
```